apps/gitness/package.json

```diff
@@ -24,6 +24,7 @@
     "@harnessio/code-service-client": "3.5.0",
     "@harnessio/forms": "workspace:*",
     "@harnessio/pipeline-graph": "workspace:*",
+    "@harnessio/core-design-system": "workspace:*",
```

Comment:
You can remove this. Gitness doesn't use the package directly, only `ui` should.